lib/signs_ui/refresh_token_store.ex

```diff
@@ -0,0 +1,50 @@
+defmodule SignsUi.RefreshTokenStore do
+  @moduledoc """
+  Stores refresh tokens for user authentication. Currently these are
+  just kept in memory because there's only one instance of the
+  application running and deploys / restarts are rare, but if need be
+  the data could be moved into persistent storage in the future.
+  """
+
+  use GenServer
+
+  @type state :: %{
+          String.t() => String.t()
+        }
+
+  def start_link(opts \\ []) do
+    name = opts[:gen_server_name] || __MODULE__
+    GenServer.start_link(__MODULE__, %{}, name: name)
```

Comment:
Since this is basically a simple key-value store, I wonder if it's any clearer if this module is implemented as an `Agent` (or ETS, I guess)?

Reply:
Huh, I briefly considered ETS but I haven't actually used Agent before, so it didn't occur to me. I'll look into it (or maybe just use ETS, honestly, since I'm already familiar with it).

Reply:
I ended up using Agent, it made the code somewhat simpler.